beacon-chain/monitor/process_block_test.go

```diff
@@ -45,7 +45,7 @@ func TestProcessSlashings(t *testing.T) {
 					},
 				},
 			},
-			wantedErr: "\"Proposer slashing was included\" ProposerIndex=2",
+			wantedErr: "\"Proposer slashing was included\" BodyRoot1= BodyRoot2= ProposerIndex=2 SlashingSlot=33 Slot=0",
```

Comment:
```suggestion
			wantedErr: "\"Proposer slashing was included\" BodyRoot1= BodyRoot2= ProposerIndex=2",
```
or make 33 equal to `params.BeaconConfig().SlotsPerEpoch + 1` cause this may eventually change in other networks.